__tests__/app.test.js

```diff
@@ -34,9 +34,42 @@ describe("/api", () => {
 			.get("/api")
 			.expect(200)
 			.then(({ body }) => {
-				console.log(body)
 				expect(body.endpoints).toEqual(endpoints);
 			});
 	});
 
 });
+
+describe("/api/articles/:article_id", () => {
+	it("GET: 200 - responds with a correct article object", () => {
+		return request(app)
+			.get("/api/articles/3")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.article.author).toBe("icellusedkars");
+				expect(body.article.title).toBe("Eight pug gifs that remind me of mitch");
+				expect(body.article.body).toBe("some gifs");
+				expect(body.article.topic).toBe("mitch");
+				expect(body.article.created_at).toBe("2020-11-03T09:12:00.000Z");
+				expect(body.article.votes).toBe(0);
+				expect(body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
```

Comment:
You have tested for a lot of properties but forgot the most important one! The article id! Please do add this

Reply:
My bad. I thought it is not necessary when id is a search parameter. I'll sort it out :)